src/runtime-rs/crates/hypervisor/src/qemu/inner.rs

```diff
@@ -402,9 +564,14 @@ impl QemuInner {
         // Confidential Guest doesn't permit virtio-fs.
         let flags = if self.hypervisor_config().security_info.confidential_guest
             || self.hypervisor_config().shared_fs.shared_fs.is_none()
-        {
+            || matches!(
```

Comment:
This issue might be addressed via updating your code, it has been addressed months ago.